packages/studio/src/components/ComponentNode.tsx

```diff
@@ -79,7 +79,7 @@ export default function ComponentNode(props: ComponentNodeProps): JSX.Element {
   return (
     <div className={componentNodeClasses} style={componentNodeStyle}>
       <div
-        className="flex grow items-center cursor-pointer"
+        className="flex items-center cursor-pointer"
```

Comment:
Do we need to remove this `grow`?  I without it, we will need to click the name of the component directly, instead of anywhere in that selection "bar". Will send a loom shortly.